src/app/components/ChartbeatAnalytics/utils/index.test.ts

```diff
@@ -291,243 +284,540 @@ describe('Chartbeat utilities', () => {
+          title: 'Media Article with video Page title',
+        };
 
-        const mockTitle = jest.fn().mockImplementation(() => pageTitle);
+        const expectedConfig = {
+          domain: 'test.bbc.co.uk',
+          idSync: {
+            bbc_hid: 'foobar',
+          },
+          path: '/',
+          sections: 'Pidgin, Pidgin - video',
+          title: 'Media Article with video Page title',
+          type: 'article-sfv',
+          uid: 50924,
+          useCanonical: true,
+          virtualReferrer: null,
+        };
 
-        if (pageType === ARTICLE_PAGE) {
-          // @ts-expect-error requires mocking for testing purposes
-          articleUtils.getPromoHeadline = mockTitle;
-        } else {
-          // @ts-expect-error requires mocking for testing purposes
-          frontPageUtils.getPageTitle = mockTitle;
-        }
+        expect(getConfig(fixtureData)).toStrictEqual(expectedConfig);
+      });
+
+      it("should have 'audio' in its identifier when the primary media type is audio", () => {
+        // @ts-expect-error testing partial data to ensure behaviour is as expected
+        const fixtureData: GetConfigProps = {
+          pageType: MEDIA_ARTICLE_PAGE,
+          taggings: [
+            {
+              predicate: 'http://www.bbc.co.uk/ontologies/bbc/infoClass',
+              value:
+                'http://www.bbc.co.uk/things/0db2b959-cbf8-4661-965f-050974a69bb5#id',
+            },
+            {
+              predicate: 'http://www.bbc.co.uk/ontologies/bbc/primaryMediaType',
+              value:
+                'http://www.bbc.co.uk/things/fe1fbc8a-bb44-4bf8-8b12-52e58c6345a4#id',
+            },
+          ],
+          service: 'pidgin',
+          title: 'Media Article with audio page title',
+        };
 
-        expect(getTitle({ pageType, pageData, brandName })).toBe(pageTitle);
+        const expectedConfig = {
+          domain: 'test.bbc.co.uk',
+          idSync: {
+            bbc_hid: 'foobar',
+          },
+          path: '/',
+          sections: 'Pidgin, Pidgin - audio',
+          title: 'Media Article with audio page title',
+          type: 'article-sfv',
+          uid: 50924,
+          useCanonical: true,
+          virtualReferrer: null,
+        };
 
-        expect(mockTitle).toHaveBeenCalledTimes(expectedNumberOfCalls);
-      },
-    );
+        expect(getConfig(fixtureData)).toStrictEqual(expectedConfig);
+      });
 
-    test.each`
-      pageType      | context               | pageTitle
-      ${MEDIA_PAGE} | ${'(onDemand TV)'}    | ${'OnDemand TV Page Title'}
-      ${MEDIA_PAGE} | ${'(onDemand Radio)'} | ${'OnDemand TV Radio Title'}
-      ${MEDIA_PAGE} | ${'(Live Radio)'}     | ${'Live Radio Title'}
-    `(
-      'should return correct title when pageType is $pageType $context',
-      ({ pageType, pageTitle }) => {
-        const pageData = {
-          pageTitle,
+      it("should have 'article-sfv' in its identifier when there are no taggings", () => {
```

Comment:
What is it meant by no taggings?

Reply:
Taggings are in the `metadata.passport.taggings` object for (optimo?) articles, and based on the values of these taggings, will determine if the content is audio or video. If there are no taggings, I believe the default identifier is `article-sfv`

https://github.com/bbc/simorgh/blob/70ef6ad00c1eca018ab7b56bc82faaf016ecc14a/data/afrique/articles/c7yn6nznljdo.json#L1757-L1758